x/marker/simulation/proposals.go

```diff
@@ -79,18 +79,18 @@ func SimulateCreateSupplyIncreaseProposalContent(k keeper.Keeper) simtypes.Conte
 			return nil
 		}
 
-		newSupply := randomUint64(r, k.GetMaxTotalSupply(ctx)-k.CurrentCirculation(ctx, m).Uint64())
+		newSupply := randomUint64(r, k.GetMaxSupply(ctx).Uint64()-k.CurrentCirculation(ctx, m).Uint64())
```

Comment:
Since we are now allowing more than uint64 in max supply these tests need to be modified to extend coverage to the higher amounts.